java/pom.xml

```diff
@@ -24,6 +24,10 @@
           <version>2.5.3</version>
         <configuration>
           <archive>
+            <manifestEntries>
+              <!-- FIXME replace logo.png with high quality splash image. -->
+              <SplashScreen-Image>logo.png</SplashScreen-Image>
```

Comment:
@i-make-robots this splash screen image feature works when you use ```mvn package```, in order for Eclipse's File -> Export to take advantage of this manifest entry you'd have to manually put it in or somehow configure Eclipse to do it. I don't suggest the former and don't know how to do the latter.